code/__DEFINES/projectiles.dm

```diff
@@ -73,3 +73,9 @@
 
 #define RETURN_POINT_VECTOR(ATOM, ANGLE, SPEED) (new /datum/point/vector(ATOM, null, null, null, null, ANGLE, SPEED))
 #define RETURN_POINT_VECTOR_INCREMENT(ATOM, ANGLE, SPEED, AMT) (new /datum/point/vector(ATOM, null, null, null, null, ANGLE, SPEED, AMT))
+
+/// The amount of energy that a standard energy weapon cell can hold
+#define STANDARD_CELL_CHARGE 1000
```

Comment:
I should have addressed this earlier, but I've a small gripe on that this define is not being used in the standard cell path at all.

Reply:
All the cells in cell.dm use the define for max charge and charge rate. I don't think there are any other cells hidden around in other files.